tools/config_updater.py

```diff
@@ -501,6 +501,18 @@ def ConfigureDatastore(config):
     config.Set("Datastore.location", datastore_location)
 
   if datastore == "2":
+    print """\n\n***WARNING***
+
+Do not continue until a MySQL server, version 5.6 or greater, is running and a user
+with the ability to create the GRR database and tables has been created. You will need
+the server, username, password, and database name (if already created) to continue. If
+no database has been created this script will attempt to create the necessary database 
+and tables using the credentials provided.
+
+***WARNING***
+"""
```

Comment:
Looks like it isn't wrapped at 80 correctly, there's breaks in the middle of sentences